contracts/utils/GSNTypes.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.5.16;
+
+library GSNTypes {
+
+    struct GasData {
+        uint256 gasLimit;
+        uint256 gasPrice;
+        uint256 pctRelayFee;
+        uint256 baseRelayFee;
```

Comment:
I didnt' see anywhere you use the `baseRelayFee`

Reply:
It is not yet supported. But I was already changing the signature format, so...